src/accessibility-statement.md

```diff
@@ -1,6 +1,83 @@
 ---
-title: Accessibility statement
+title: Accessibility statement for the GOV.UK Brand Guidelines website
 excludeFromNavigation: true
 ---
 
-[TODO]: Fill content in
+This accessibility statement applies to govuk-brand-guidelines.service.gov.uk.
+
+This website is run by the Government Digital Service (GDS). We want as many people as possible to be able to use this website. For example, that means you should be able to:
+
+- change colours, contrast levels and fonts using browser or device settings
+- zoom in up to 400% without the text spilling off the screen
+- navigate most of the website using a keyboard or speech recognition software
+- listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
+
+AbilityNet has advice on [making your device easier to use](https://mcmw.abilitynet.org.uk/) if you have a disability.
+
+## How accessible this website is
+
+We know some parts of this website are not fully accessible:
+
+- Some videos don't have a description and don't say they have no audio.
+- Sets of colours are missing clear groupings.
```

Comment:
Style guide corrections for bullet point style, and negative contractions.

```suggestion
- some videos do not have a description and don't say they have no audio
- sets of colours are missing clear groupings
```

Reply:
Oops, ignore me. I was looking at the wrong line. All good.